website/src/app/views/our-team/members.js

```diff
@@ -182,6 +183,12 @@ const members = {
         linkedinURL: 'https://www.linkedin.com/in/carlosalim7/',
         imageURL: carlos,
       },
+      {
+        name: 'Emmy Chen',
+        position: 'Developer',
+        linkedinURL: 'www.linkedin.com/in/emmy-chen-583552252',
```

Comment:
This need an `https://` before the rest of the URL for it to direct to an external site like LinkedIn; otherwise, HTML thinks you're trying to go to a different page on the same website